api/src/main/resources/config/application.properties

```diff
@@ -46,6 +46,8 @@ app.baseDomain=osmt.wgu.edu
 app.baseUrl=https://${app.baseDomain}
 app.frontendUrl=https://${app.baseDomain}
 app.loginSuccessRedirectUrl=${app.frontendUrl}/login/success
+app.userName=name
```

Comment:
It's good to add a comment here describing that these are strings identifying JWT claims.

Reply:
yup! sounds reasonable!